app/platform/templates/platform/developers/add_form.html

```diff
@@ -0,0 +1,35 @@
+{% extends "platform/manage_grant_base.html" %}
+
+{% from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+
+
+{% block pageTitle %}
+    Add a form - {{ section.title }} - MHCLG Funding Service
+{% endblock pageTitle %}
+
+{% set active_sub_navigation_tab = "grant_developers" %}
+
+{% block beforeContent %}
+    {{ govukBackLink({
+        "text": "Back",
+        "href": url_for("platform.add_form", grant_id = grant.id, collection_id = collection.id, section_id=section.id)
+    }) }}
+{% endblock beforeContent %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">What is the name of the form?</h1>
+            <form method="post" novalidate>
+                {{ form.csrf_token }}
+                {{ form.collection_id }}
+                {{ form.title }}
+                {{ form.submit(params={
+                    "text": "Add form"
+                }) }}
+            </form>
```

Comment:
This looks a bit odd and duplicative/redundant

<img width="545" alt="image" src="https://github.com/user-attachments/assets/9298b477-adc2-4881-b7cc-f04c4e7b1608" />

I'm used to seeing the question title declared on the form (we currently have placeholder-feeling text of "Form title"; if we change that to the question itself of "What is the name of the form?" then we can pass params to form.title to render the field with the question as an h1 based on the template options (nunjucks here, but the equivalent jinja works):

<img width="863" alt="image" src="https://github.com/user-attachments/assets/06a915ef-103c-4cac-b200-5944cde834e9" />